src/tools/scripts/incoherenceStructure/structure_sans_coselec.js

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env node
+'use strict';
+
+const { execute } = require('../../utils');
+const path = require('path');
+const fs = require('fs');
+
+const getStructures = async db => await db.collection('structures').find({
+  'statut': { '$ne': 'VALIDATION_COSELEC' }
+}).toArray();
+
+const countUserStructure = db => async idStructure => await db.collection('users').countDocuments({
+  'entity.$id': idStructure
+});
+
+const countStatutsIncorrects = db => async idStructure => await db.collection('misesEnRelation').aggregate(
+  { '$match': {
+    'structure.$id': idStructure,
+    'statut': { '$ne': 'finalisee_rupture' }
+  } },
+  { '$group': {
+    '_id': '$statut',
+    'countStatut': { '$sum': 1 }
+  } },
+  { '$project': {
+    '_id': '$_id',
+    'countStatut': '$countStatut'
+  } }
+).toArray();
+
+const getMisesEnRelation = db => async idStructure => await db.collection('misesEnRelation').find({
+  'structure.$id': idStructure,
+  'statut': 'finalisee_rupture'
```

Comment:
`$in: ['finalisee_rupture', 'terminee']`

Reply:
ah oui ok dissocié ici, si tu laisses que la première requete en supprimant le $in, tu devrais avoir direct dans le $group tout